lib/cbus_elixir/app/meetings.ex

```diff
@@ -0,0 +1,31 @@
+defmodule CbusElixir.App.Meetings do
+  @moduledoc """
+  Domain operations for meetings.
+  """
+
+  import Ecto.Query, warn: false
+  alias CbusElixir.Repo
+
+  alias CbusElixir.App.Meeting
+
+  @doc """
+  Returns the next 5 upcoming meetings
+  """
+  def upcoming_meetings_for(date) do
```

Comment:
next_5_meetings?

or

```elixir

def upcoming_meetings_for(date, n) do
  ... Find the next n meetings
end
def upcoming_meetings_for(date, _) do
  ... n = 5
end
```

Reply:
This method name is confusing. Why not call is `next_five_meetings(date)` or `meetings_after_date(date)`